dnf.spec

```diff
@@ -38,7 +38,7 @@
 %endif
 
 # configurable name for the compat yum package
-%global yum_subpackage_name %{name}-yum
+%global yum_subpackage_name yum
```

Comment:
Is this change of yum_subpackage_name intentional? I'm afraid we can't do this for Fedoras.

Reply:
Sorry, this change was only for testing